sp800-63a/sec9_usability.md

```diff
@@ -74,21 +76,21 @@ Usability considerations for the enrollment session include:
 *   If at all possible, avoid using KBV, as KBV is extremely problematic from a usability perspective. It is error-prone and frustrating for users given the limitations of human memory, and has a low probability of success. 
     * The types of questions used in KBV are often extremely difficult for users to answer correctly. The questions often ask users to recall old, infrequently used information, or information that changes over time. KBV questions must still have relevance to users for them to be able to answer correctly. 
-    * It is especially important that KBV questions are clearly phrased, as ambiguity can lead to user errors when answering. For example, if the KBV question is asking about a user’s remaining mortgage balance, clearly specify which mortgage because users may have multiple mortgages. This question can be phrased as “What is the remaining mortgage balance on your primary residence?” to reduce ambiguity without compromising the integrity of the proofing process and without revealing unnecessary details about the individual. 
-    * Free-form versus multiple-choice KBV questions have different usability implications. If free-form responses are required, accept all semantically correct answers. For example, accept all aliases for a user’s school name: full name, short name, and widely used nickname. Don’t make acceptable responses case-sensitive, and do ignore the use of pronouns such as ‘the’ or ‘a’.
+    * It is especially important that CSPs clearly phrase KBV questions, as ambiguity can lead to user errors when answering. For example, if the KBV question is asking about a user’s remaining mortgage balance, clearly specify which mortgage because users may have multiple mortgages. This question can be phrased as “What is the remaining mortgage balance on your primary residence?” to reduce ambiguity without compromising the integrity of the proofing process and without revealing unnecessary details about the individual. **MG: we need references to the KBV section here as some of the example could reveal private information. I'm queasy having KBV guidance here at all.**
+    * Free-form versus multiple-choice KBV questions have different usability implications. If free-form responses are required, accept all semantically correct answers. For example, accept all aliases for a user’s school name: full name, short name, and widely used nickname. Don’t make acceptable responses case-sensitive, and do ignore the use of pronouns such as ‘the’ or ‘a’. **MG: what is school name?**
```

Comment:
@jimfenton good catch, even though it's an example.  Please provide this comment in your offline word version.